vllm/model_executor/models/gpt_bigcode.py

```diff
@@ -260,15 +272,35 @@ def __init__(
         self.transformer = GPTBigCodeModel(config, cache_config, quant_config,
                                            lora_config)
         if self.config.tie_word_embeddings:
-            self.lm_head = self.transformer.wte
+            if quant_config is not None:
+                linear_method = quant_config.get_quant_method(self)
+                assert linear_method is None or isinstance(
+                    linear_method, UnquantizedLinearMethod)
+
+            self.lm_head = ParallelLMHead(
+                self.unpadded_vocab_size,
```

Comment:
Umm should line 300 be before this where we declare `self.unpadded_vocab_size`?

Reply:
Yes, my bad.